resource-definitions/terraform-driver/backends/s3/s3-backend.yaml

```
@@ -0,0 +1,39 @@
+apiVersion: entity.humanitec.io/v1b1
+kind: Definition
+metadata:
+  id: example-terraform-s3-backend-s3
+entity:
+  # Update based on your AWS Cloud Account
+  driver_account: aws-account
+  driver_inputs:
+    values:
+      credentials_config:
+        environment:
+          AWS_ACCESS_KEY_ID: "AccessKeyId"
+          AWS_SECRET_ACCESS_KEY: "SecretAccessKey"
+          AWS_SESSION_TOKEN: "SessionToken"
+      script: |
+        variable "region" {}
+
+        terraform {
+          backend {
```

Doesn't the `backend` block have to name its type? See https://developer.hashicorp.com/terraform/language/backend/s3
```suggestion
          backend "s3" {
```